spec/scorecardSpec.js

```diff
@@ -0,0 +1,189 @@
+describe('Scorecard', function() {
+
+  var scorecard;
+
+  beforeEach(function() {
+    scorecard = new Scorecard(Frame);
+  });
+
+  describe('when initialised', function() {
+
+    it('has ten frames', function() {
+      expect(scorecard.frameRecord).toEqual([new Frame, new Frame, new Frame, new Frame, new Frame, new Frame, new Frame, new Frame, new Frame, new Frame]);
```

Comment:
How would you have tested 100 frames?  In order to verify this test is correct, I have to check that there are 10 `new Frame` entries in the expectation.  This makes the test unreadable and easily mistaken.  You should perhaps have 2 tests - one that tests there are 10 (using length) and another that each item is indeed a `Frame` (if you want to go that far).